packages/web/docs/src/pages/product-updates/2024-06-14-ms-teams-webhooks-alerts.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: MS Teams Webhooks Alerts
+description: Configure MS Teams webhooks to receive alerts and notifications from Hive.
+date: 2024-06-14
+authors: [capaj]
```

Comment:
You probably need to add yourself to the list of authors

Reply:
added, thanks for the pointer, would have taken me hours to figure it out.